configure.ac

```diff
@@ -45,7 +45,7 @@ AC_ARG_WITH(
     [ CRM_DAEMON_USER="hacluster" ],
   )
 
-AM_INIT_AUTOMAKE([no-define])
+AM_INIT_AUTOMAKE([foreign])
```

Comment:
`foreign` allows the the error surrounding a lack of a plain `README` file with no file extension to be suppressed.